source/_integrations/eway.markdown

```
@@ -0,0 +1,193 @@
+    description: "Password for MQTT authentication (optional)."
+Device ID:
+    description: "Your Eway device identifier."
+Device Serial Number:
+    description: "Your device's serial number."
+Device Model:
+    description: "Your device's model code (optional, default: Unknown)."
+Scan Interval:
+    description: "Data update interval in seconds (default: 30)."
+{% endconfiguration_basic %}
+
+# Eway Home Assistant Integration
```

markdownlint-cli2 (0.17.2)

No space after hash on atx style heading

(MD018, no-missing-space-atx)